recipes/nmma/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set name = "nmma" %}
+{% set version = "0.0.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/nmma-{{ version }}.tar.gz
+  sha256: 9394a212c06a60e06181b2f0b78e8015d41280aac89dbc6ab19156c6f09ab3fa
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools
+  run:
+    - arviz
+    - astropy >=4.3.1
+    - bilby >=1.2.0
+    - bilby_pipe ==1.0.5
+    - dust_extinction
+    - future
+    - matplotlib-base >=2.0
+    - numpy >=1.9
+    - pandas >=1.3.4
+    - pymultinest
+    - python
+    - scikit-learn >=0.23.1
+    - scipy >=1.7.1
+    - sncosmo
+
+test:
+  imports:
+    - nmma
+  commands:
+    - pip check
```

You may need to adjust the entrypoints tests if there is no help command to check them.

```suggestion
    - pip check
    - nmma_analysis --help
    - nmma_generation --help
    - parallel_em_analysis --help
    - parallel_em_generation --help
    - light_curve_analysis --help
    - light_curve_injection_summary --help
    - light_curve_injection_slurm_setup --help
    - light_curve_injection_condor_setup --help
    - lightcurve_marginalization --help
    - combine_EOS --help
    - create_light_curve_slurm --help
    - create_light_curve_condor --help
    - create_svdmodel --help
    - light_curve_generation --help
    - light_curve_detection --help
    - nmma_create_injection --help
    - gwem_resampling --help
    - gwem_resampling_condor --help
    - gwem_Hubble_estimate --help
    - light_curve_analysis_condor --help
```